src/views/Admin.js

```diff
@@ -1,84 +1,67 @@
-import React, { useCallback } from 'react';
-import { Just } from 'folktale/maybe';
+import React from 'react';
 import { Grid } from 'indigo-react';
-import * as need from 'lib/need';
-
-import View from 'components/View';
-import { ForwardButton } from 'components/Buttons';
-
-import { ROUTE_NAMES } from 'lib/routeNames';
-import { eqAddr } from 'lib/wallet';
 
 import { useHistory } from 'store/history';
-import { useWallet } from 'store/wallet';
-import { usePointCursor } from 'store/pointCursor';
-import { usePointCache } from 'store/pointCache';
-import FooterButton from 'components/FooterButton';
 
-export default function Admin() {
-  const history = useHistory();
-  const { urbitWallet, wallet } = useWallet();
-  const { pointCursor } = usePointCursor();
-  const { pointCache } = usePointCache();
+import useCurrentPointName from 'lib/useCurrentPointName';
+import useRouter from 'lib/useRouter';
+import { LocalRouterProvider } from 'lib/LocalRouter';
 
-  const point = need.point(pointCursor);
-  const pointDetails = need.fromPointCache(pointCache, point);
-  const address = need.addressFromWallet(wallet);
+import View from 'components/View';
+import Crumbs from 'components/Crumbs';
 
-  const goRedownload = useCallback(() => history.push(ROUTE_NAMES.REDOWNLOAD), [
-    history,
-  ]);
+import AdminHome from './Admin/AdminHome';
+import AdminEditPermissions from './Admin/AdminEditPermissions';
+import AdminRedownload from './Admin/AdminRedownload';
+import AdminReticket from './Admin/AdminReticket';
+import AdminSetProxy from './Admin/AdminSetProxy';
```

Comment:
I'd personally not prefix these with `Admin` if they're already in the `/Admin` folder, but I guess I can see how this is nice for identifiability by just bare filename. They're still unique names regardless though.

Reply:
Also, in the future, might be good to `git mv` the renaming before/after making changes, so that there's a legible diff rather than completely "new" files. Looks like it did pick up on that for _some_ files, but wasn't smart enough to catch them all.

Reply:
totally - I prefix them so that fuzzy search in editors works better and the chances of a collision between folders (like what could happen with `Activate/Passport` and `components/Passport`) to avoid confusion. I don't feel _super_ strongly about this though, so I'm willing to revert that pattern. lmk